arrowfunctions.js

```diff
@@ -0,0 +1,7 @@
+console.log("Program on arrow functions")
```

Comment:
practice the single line return function

Reply:
Okay